docs-ref-conceptual/includes/cli-install-linux-apt.md

```diff
@@ -130,13 +132,13 @@ You can also use `apt-get upgrade` to update the CLI package. This command upgra
 2. If you don't plan to reinstall the CLI, remove the Azure CLI repository information:
 
    ```bash
-   sudo rm /etc/apt/sources.list.d/azure-cli.list
+   sudo rm /etc/apt/sources.list.d/azure-cli.sources
    ```
 
 3. If you aren't using other packages from Microsoft, remove the signing key:
 
     ```bash
-    sudo rm /etc/apt/trusted.gpg.d/microsoft.gpg
+    sudo rm /etc/apt/keyrings/microsoft.gpg
```

Comment:
This change should have been done in https://github.com/MicrosoftDocs/azure-docs-cli/pull/3435.

Reply:
But it wasn't?